inst/style-sets/anvil/.github/workflows/delete-preview.yml

```diff
@@ -0,0 +1,25 @@
+name: Delete Preview
```

Comment:
I don't think there will be many updates to the main template and even if we do make any updates, users can fetch the updates by downloading the development version of ottrpal from GitHub.  

The main template is https://github.com/fhdsl/ottr. It is not https://github.com/jhudsl/OTTR_Template/, since this function setups up the Quarto-based OTTR template, not the Rmd-based OTTR Template.

Reply:
There will definitely be more updates to OTTR in the future. We should also probably have this try to work for both Rmds and Quarto if possible (but I know that's another lift). 

Some of the files you are having it download are GHA which will definitely change because people's GHA that we are using deprecate overtime. I think downloading the files directly will be a more stable move but we can definitely brainstorm on this.

Reply:
Yeah I'll setup a meeting with you to talk about this further.